analyses/molecular-subtyping-HGG/results/HGG_defining_lesions.tsv

```diff
@@ -213,7 +213,6 @@ PT_AW8WV14Y	7316-179	BS_YYAPSA5P	No	No	No	No	No	No	FALSE
 PT_AXMVG8QE	7316-2732	BS_Q88059DJ	No	No	No	No	No	No	FALSE
 PT_AXVH36AM	7316-2180	BS_ZPF89720	No	No	No	No	No	No	FALSE
 PT_AYZA2WCW	7316-197	BS_0Q8G4N8H	No	No	No	No	No	No	FALSE
-PT_B0JFCN5W	7316-1773	BS_BZ4GKDP4	No	No	No	No	No	No	FALSE
```

Comment:
Since the code generating HGG_defining_lesions.tsv is generated by checking for histone variant status in all samples in the consensus maf, only samples in the consensus maf get a status. 

In v18, consensus maf had an intron variant: 
  | Hugo_Symbol | Entrez_Gene_Id | Center | NCBI_Build | Chromosome | Start_Position | End_Position | Strand | Variant_Classification | Variant_Type | Reference_Allele | Tumor_Seq_Allele1 | Tumor_Seq_Allele2 | dbSNP_RS | dbSNP_Val_Status | Tumor_Sample_Barcode | Matched_Norm_Sample_Barcode | Match_Norm_Seq_Allele1 | Match_Norm_Seq_Allele2 | Tumor_Validation_Allele1 | Tumor_Validation_Allele2 | Match_Norm_Validation_Allele1 | Match_Norm_Validation_Allele2 | Verification_Status | Validation_Status | Mutation_Status | Sequencing_Phase | Sequence_Source | Validation_Method | Score | BAM_File | Sequencer | Tumor_Sample_UUID | Matched_Norm_Sample_UUID | HGVSc | HGVSp | HGVSp_Short | Transcript_ID | Exon_Number | t_depth | t_ref_count | t_alt_count | n_depth | n_ref_count | n_alt_count | all_effects | Allele | Gene | Feature | Feature_type
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
OR2T35 | 403244 | . | GRCh38 | chr1 | 248645014 | 248645015 | + | Intron | DEL | TG | TG | - | rs66799644 | . | BS_BZ4GKDP4 | BS_YHZ69SDZ | TG | TG | . | . | . | . | . | . | . | . | . | . | . | . | . | . | . | c.-23+232_-23+233del | . | . | ENST00000641268 | . | 68 | 58 | 5 | 36 | 34 | 0 | OR2T35,intron_variant,,ENST00000641268,;OR2T27,downstream_gene_variant,,ENST00000344889,NM_001001824.1;OR2T27,downstream_gene_variant,,ENST00000460972,;OR2T27,downstream_gene_variant,,ENST00000641652,; | - | ENSG00000177151 | ENST00000641268 | Transcript

In v19, we don't have any consensus variant for this sample, the variant above is probably being removed because higher gnomad AF  https://www.ncbi.nlm.nih.gov/snp/rs66799644. I will report back after I check for this sample in the newly rerun consensus file.

Reply:
I just wanted to follow up that there are variant calls in the individual variant files so this sample is included the raw files in v19 as well, just none of the variant calls were retained in the consensus calls.

Reply:
I am confused about this one because the gene listed above (OR2T35) and the one linked above (OR2T10) are not the histone genes. What was the defining lesion before?

Reply:
The code generating HGG_defining_lesions.tsv is generated by

- checking for histone variant status in **ALL** Tumor_Sample_Barcode **in** the consensus maf
- removing all LGAT samples since LGG with Histone variant would be a subtype of LGG only

Since the HGG defining code uses the Tumor_Sample_Barcode column so only DNA samples in the consensus maf file will get a HGG defining status [in this code](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/fcad2e5681a716b15d202a9e95a0b855246f97de/analyses/molecular-subtyping-HGG/01-HGG-molecular-subtyping-defining-lesions.Rmd#L94)

I checked `BS_BZ4GKDP4` is not in the v19 consensus maf file so will not be in the HGG_defining_lesions.tsv file.

 In v18 only  the following variant was found ( which you are right is not histone) but because this variant existed this sample was in the v18 consensu maf file and in turn existed in HGG_defining_lesions.tsv file.
 
  | Hugo_Symbol | Entrez_Gene_Id | Center | NCBI_Build | Chromosome | Start_Position | End_Position | Strand | Variant_Classification | Variant_Type | Reference_Allele | Tumor_Seq_Allele1 | Tumor_Seq_Allele2 | dbSNP_RS | dbSNP_Val_Status | Tumor_Sample_Barcode | Matched_Norm_Sample_Barcode | Match_Norm_Seq_Allele1 | Match_Norm_Seq_Allele2 | Tumor_Validation_Allele1 | Tumor_Validation_Allele2 | Match_Norm_Validation_Allele1 | Match_Norm_Validation_Allele2 | Verification_Status | Validation_Status | Mutation_Status | Sequencing_Phase | Sequence_Source | Validation_Method | Score | BAM_File | Sequencer | Tumor_Sample_UUID | Matched_Norm_Sample_UUID | HGVSc | HGVSp | HGVSp_Short | Transcript_ID | Exon_Number | t_depth | t_ref_count | t_alt_count | n_depth | n_ref_count | n_alt_count | all_effects | Allele | Gene | Feature | Feature_type
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
OR2T35 | 403244 | . | GRCh38 | chr1 | 248645014 | 248645015 | + | Intron | DEL | TG | TG | - | rs66799644 | . | BS_BZ4GKDP4 | BS_YHZ69SDZ | TG | TG | . | . | . | . | . | . | . | . | . | . | . | . | . | . | . | c.-23+232_-23+233del | . | . | ENST00000641268 | . | 68 | 58 | 5 | 36 | 34 | 0 | OR2T35,intron_variant,,ENST00000641268,;OR2T27,downstream_gene_variant,,ENST00000344889,NM_001001824.1;OR2T27,downstream_gene_variant,,ENST00000460972,;OR2T27,downstream_gene_variant,,ENST00000641652,; | - | ENSG00000177151 | ENST00000641268 | Transcript

Then to identify why this variant (rs66799644) is no longer in the consensus maf file I checked if the gnomad AF  ,  https://www.ncbi.nlm.nih.gov/snp/rs66799644 which is more than 0.001 which is now a criteria for germline-masking. [reference to bix dev description](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/990#issuecomment-824948640)

Reply:
Ah, I see - because it was in the MAF _at all_, it got captured before and now is not in the MAF. I misunderstood - thank you!